packages/react-components/react-persona/Spec.md

```diff
@@ -1,63 +1,306 @@
 
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+_PersonaIcon vs PersonaCoin_: PersonaCoin allows you to have icon only when there's no image available or it's the unknown PersonaCoin variant. In v9's PersonaIcon, you have the option of showcasing an icon through the Avatar or only the icon.
+
+- Note: v8 Persona only shows and icon if there's no presence and size is `tiny`, `size8`, or `size16`.
+
+_PersonaPresenceBadge vs PersonaCoin_: PersonaCoin allows you to have only a PresenceBadge when the size of the coin is `tiny`, `size8`, or `size16`. In v9's Persona you are able to render the PersonaPresenceBadge alone.
+
+_Image vs PersonaCoin_: PersonaCoin allows you to have an image only within the Avatar. This can be avoided by rendering a custom coin, but there's still all the dom and styling overhead that comes with PersonaCoin. v9's Persona allows you use a single image and does not resize the image.
+
+### Text lines
+
+Other than styling and naming, the text lines in Persona remain the same.
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
+![Avatar](./etc/images/avatar.png)
+
+Persona with Avatar:
+
+```jsx
+<Persona media={<PersonaAvatar name="Kevin Sturgis" />} primaryText="Kevin Sturgis" secondaryText="Software Engineer" />
+```
+
+![Avatar and PresenceBadge](./etc/images/avatar_badge.png)
+
+Persona with Avatar + PresenceBadge:
+
+```jsx
+<Persona
+  media={<PersonaAvatar name="Kevin Sturgis" badge={{ status: 'offline', outOfOffice: true }} />}
+  primaryText="Kevin Sturgis"
+  secondaryText="Software Engineer"
+  tertiaryText="Offline"
+/>
+```
+
+![Icon](./etc/images/icon.png)
+
+Persona with icon:
+
+```jsx
+<Persona
+  media={
+    <PersonaIcon>
+      <PersonCallRegular />
+    </PersonaIcon>
+  }
+  primaryText="Person Call Icon"
+/>
+```
+
+![PresenceBadge](./etc/images/badge.png)
+
+Persona with PresenceBadge:
+
+```jsx
+<Persona media={<PersonaPresenceBadge status="offline" outOfOffice={true} />} primaryText="Kevin Sturgis" />
+```
+
+![Image](./etc/images/image.png)
+
+Persona with image:
+
+```jsx
+<Persona
+  primaryText="Katri Athokas"
+  secondaryText="Software Engineer"
+  tertiaryText="CXE"
+  quaternaryText="Offline"
+  image={<img src="https://fabricweb.azureedge.net/fabric-website/assets/images/avatar/KatriAthokas.jpg" />}
+/>
+```
 
 ## Variants
 
-_Describe visual or functional variants of this control, if applicable. For example, a slider could have a 2D variant._
+There are three alignment variants:
+
+- start: Media on the left and text on the right.
+- center: Media on top and text on the bottom.
+- end: Media on the right and text on the left.
+
+There are 4 Media variants:
+
+- PersonaPresenceBadge
+- PersonaIcon
+- PersonaAvatar
```

Comment:
Can you please clarify what these components do and why they are required? For example, why I cannot do `<Persona media={<BookIcon />}>`?

Reply:
Done, thanks for pointing it out!